serving/docker/partition/partition.py

```diff
@@ -121,23 +121,25 @@ def download_config_from_hf(self):
         return downloaded_dir
 
     def copy_config_files(self):
-        model_dir = self.properties['model_dir']
-        if 'model_id' in self.properties:
+        model_dir = self.properties['option.model_dir']
+        if 'option.model_id' in self.properties:
             model_dir = self.download_config_from_hf()
 
         config_files = []
         for pattern in CONFIG_FILES_PATTERNS:
             config_files += glob.glob(os.path.join(model_dir, pattern))
 
         for file in config_files:
-            shutil.copy(file, dst=self.properties['save_mp_checkpoint_path'])
+            shutil.copy(file,
+                        dst=self.properties['option.save_mp_checkpoint_path'])
 
     def upload_checkpoints_to_s3(self):
-        if 'upload_checkpoints_s3url' not in self.properties:
+        if 'option.upload_checkpoints_s3url' not in self.properties:
```

Comment:
No need to add 'option.` to intermediate key using only for partition

Reply:
Okay, fixed